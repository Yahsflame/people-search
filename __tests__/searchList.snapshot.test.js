import React from 'react';
import list from '../components/SearchList';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<list page="https://people-search-app.surge.sh">Versium</list>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});