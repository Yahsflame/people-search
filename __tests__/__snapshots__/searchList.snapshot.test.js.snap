// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<list
  page="https://people-search-app.surge.sh"
>
  Versium
</list>
`;
